recipes/settings.py

```diff
@@ -450,7 +450,11 @@
 
 LANGUAGE_CODE = 'en'
 
-TIME_ZONE = os.getenv('TIMEZONE') if os.getenv('TIMEZONE') else 'Europe/Berlin'
+if os.getenv('TIMEZONE') is not None:
+    print('DEPRECATION WARNING: Environment var "TIMEZONE" is deprecated. Please use "TZ" instead.')
+    TIME_ZONE = bool(int(os.getenv('REVERSE_PROXY_AUTH', False)))
```

Comment:
shouldn't this be `TIME_ZONE = bool(int(os.getenv('REVERSE_PROXY_AUTH', False)))` ?

I think `REVERSE_PROXY_AUTH` might have slipped in here

Reply:
good catch - ouch!